migrations/2024-01-22-105746_lemmynsfw-changes/up.sql

```diff
@@ -1,2 +1,9 @@
-alter table local_site add column content_warning text;
-alter table community add column only_followers_can_vote boolean not null default false;
\ No newline at end of file
+ALTER TABLE local_site
+    ADD COLUMN content_warning text;
+
+ALTER TABLE local_site
+    ADD COLUMN auto_expand_images boolean NOT NULL DEFAULT FALSE;
+
+ALTER TABLE community
+    ADD COLUMN only_followers_can_vote boolean NOT NULL DEFAULT FALSE;
```

Comment:
This one seems weird right? Esp since its trivial for anyone to follow.

Reply:
I wanted to integrate all the changes so they dont have to maintain a fork anymore. In their fork it also has a time check, so you can only vote after following the community for at least 10 minutes, but that would be too complicated to indicate in frontends.

I can also get rid of this setting.

Reply:
Yeah I really don't think its necessary. And if other forks want to maintain complicated rules around who can vote and when, that's up to them to maintain. 

Once we get private communities properly going, then we can add voting limitations.